<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>선생님! 어디가셨어요?</string>
  </property>
  <widget class="QLabel" name="teacherName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>521</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>371</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>_____________________________________________</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editStatus">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>430</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
  </widget>
  <widget class="QComboBox" name="status_comboBox">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>50</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="edit_message">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>336</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
